src/renderer/screens/account/AccountHeaderActions.js

```diff
@@ -87,6 +83,47 @@ const AccountHeaderActions = ({ account, parentAccount, openModal, t }: Props) =
     });
   }, [currency, history, mainAccount]);
 
+  // List of available exchange actions
+  const actions = [
+    ...(availableOnExchange
```

Comment:
Since there's no unified list of crypto used for all exchange features, we'll need individual checks per action (buy/swap/sell) but I'll adapt this on swap side if needed.

Reply:
For this one we might need to port the feature flagging of exchange features into live-common :+1: